### PR TITLE
Config ease of access

### PR DESCRIPTION
**Pull request by kinggoesgaming**
This is a WIP PR
- [x] Create ConfigManager. (Handles checks for NPEs)
- [x] Implement ConfigManager methods
- [x] provide a Easy setValue method for config, where values are set and save immediately to disk
- [x] Cleanup config accessor

### CODE REVIEW

**Comment by kinggoesgaming** [comment]
this PR is ready to be merged

File: src/main/java/io/github/hsyyid/essentialcmds/utils/Utils.java [hunk 1]
```diff
@@ -214,9 +207,6 @@ public static void startAFKService()
 									afk.setFood(food.foodLevel().get());
 								}
 							}
-
-							afk.setMessaged(true);
-							afk.setAFK(true);
```
Comment (hsyyid) [inline src/main/java/io/github/hsyyid/essentialcmds/utils/Utils.java:219]: Why this change?

Comment (kinggoesgaming) [inline src/main/java/io/github/hsyyid/essentialcmds/utils/Utils.java:219]: hmm i think i accidentally did that... let me undo that..

**Comment by kinggoesgaming** [comment]
check now
